data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,138 @@
+                "ltc:description": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_description A free text description or narrative about the collection.",
+                    "examples": [
+                        "A virtual collection representing all butterfly specimens in South Holland"
+                    ]
+                }
+            }
+        },
+        "ltc:MeasurementOrFact": {
+            "type": "object",
+            "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_MeasurementOrFact A measurement of or fact about a class within the standard, or a relationship between the ObjectGroup and an associated class.",
+            "properties": {
+                "dwc:measurementType": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementType The nature of the measurement, fact, characteristic, or assertion.",
+                    "examples": [
+                        "Object Count"
+                    ]
+                },
+                "dwc:measurementValue": {
+                    "type": "number",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementValue The numerical value of the measurement.",
+                    "examples": [
+                        100
+                    ]
+                },
+                "dwc:measurementUnit": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/#MeasurementOrFact_measurementUnit The units associated with the measurementValue.",
+                    "examples": [
+                        "mm, C, km, count, percentage, category"
+                    ]
+                }
+            }
+        },
+        "ods:searchFilters": {
+            "type": "array",
+            "description": "Search filters selected for the virtual collection.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "filterValue": {
+                        "type": "array",
+                        "examples": [
+                            "topicDiscipline=Zoology&midsLevel=1&country=Indonesia&hasMedia=1"
+                        ],
+                        "description": "Value or configuration of the filter."
+                    }
+                }
+            }
+        },
+        "ltc:basisOfScheme": {
+            "type": "string",
+            "description": "A summary of the basis or purpose for the LatimerCoreScheme. This property is intended to summarize the reason for grouping a number of ObjectGroups within the LatimerCoreScheme, and the purpose for which the data is intended to be used."
+        },
+        "examples": [
+            "dataset for workshop"
+        ]
+    },
+    "required": [
```

Comment:
I think searchFilters needs to be required.

Reply:
added